ga/latest/kernel/Dockerfile.ubi.ibmjava8

```diff
@@ -19,7 +19,7 @@ USER root
 ARG VERBOSE=false
 
 # Install WebSphere Liberty
-ARG LIBERTY_VERSION=24.0.0.2
+ARG LIBERTY_VERSION=24.0.0.3
 ARG LIBERTY_URL
 ARG DOWNLOAD_OPTIONS=""
 ARG LIBERTY_SHA=bd9c64cbf086f03e81ee50e4a44acc6f05af191a
```

Comment:
The SHA value must be replaced with placeholder. Same for other Dockerfiles in `latest`